src/styles/components/_sign-in-form.scss

```diff
@@ -0,0 +1,29 @@
+@import '../abstracts/_abstracts-dir';
+
+.sign-in-form {
+  display: flex;
+  flex-direction: column;
+  background-color: $background;
+
+  &__header {
+    display: flex;
+    justify-content: center;
+    background-color: greenyellow;
+    margin-bottom: rem(24px);
+    padding: rem(24px) 0;
+  }
+
+  &__body {
+    margin-bottom: rem(24px);
+  }
+
+  &__footer {
```

Comment:
Currently the `flex-direction` is row. How about adding `column` so **forgot password** stay below **Don't have an account?** 🧐